### DISCUSSION TITLE
Collector for MS sql server database health

### DISCUSSION THREAD

**Discussion by Yashwanth-6465** [burningalchemist/sql_exporter#263] [Q&A] [answered]
Hello folk

For collecting MS sql data base server health metrics should we define our own collector file...if yes can anyone share that collector file

**Comment by burningalchemist** [comment, marked as answer]
Hey @Yashwanth-6465, you're most likely have to define your own collector file. Please have a look at the existing MSSQL collector files in `examples`, so you could use it as a baseline and add other metrics you might require.
